metadata.rb

```diff
@@ -4,9 +4,11 @@
 license          "Apache 2.0"
 description      "Installs/Configures ucarp"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "1.0.2"
+version          "1.0.3"
```

Comment:
@knightsamar No, when we do a cookbook bump, this will automaitcally be changed. Its independednt of adding other changes to the metadata.